spring-data-vmware-gemfire/src/main/java/org/springframework/data/gemfire/listener/ContinuousQueryDefinition.java

```diff
@@ -4,21 +4,19 @@
  */
 package org.springframework.data.gemfire.listener;
 
-import java.lang.reflect.Method;
-import java.util.Optional;
-import java.util.function.Function;
-
-import org.apache.geode.cache.query.CqAttributes;
-import org.apache.geode.cache.query.CqAttributesFactory;
-import org.apache.geode.cache.query.CqListener;
-import org.apache.geode.cache.query.CqQuery;
-
+import org.apache.geode.cache.query.*;
```

Comment:
Formatting has gone wrong. We should not import * on any package

Reply:
Fixed